src/components/LoginRoute.js

```diff
@@ -1,4 +1,23 @@
 import { Route } from 'react-router';
 
+import Context from './../Context';
+import UserStore from './../stores/UserStore';
+
 export default class LoginRoute extends Route {
-}
\ No newline at end of file
+  static defaultProps = {
+    onEnter(nextState, replaceState, callback) {
+      UserStore.isAuthenticated((err, authenticated) => {
+        if (authenticated) {
+          var router = Context.getInstance().getRouter();
+
```

Comment:
Please remove this line to be consistent with the other code. The same for `LogoutRoute`.